mitodl/mit-open
# units page

### What are the relevant tickets?
Closes https://github.com/mitodl/hq/issues/4374

### Description (What does it do?)
This PR adds the `UnitsListingPage` page, which lists the different "units" within MIT, currently referred to in the API and models as "offerors" and represented by the `LearningResourceOfferor` model. The page shows a card for each unit which contains their logo, a "value proposition" statement and their course and program counts, separated into "academic" and "professional" by the `professional` property on the `LearningResourceOfferor` model.

### Screenshots (if appropriate):
<img width="1919" alt="image" src="https://github.com/mitodl/mit-open/assets/12089658/961c49a8-b339-4ac4-bebe-dfdd332bbe79">
<img width="220" alt="image" src="https://github.com/mitodl/mit-open/assets/12089658/82d828fe-fc17-4e20-9826-3a6ba4572027">

### How can this be tested?
 - Spin up `mit-open` on this branch
 - Update your offeror information by running `docker compose exec web ./manage.py update_offered_by`
 - Make sure you have at least some courses and programs backpopulated in your database
 - Visit http://localhost:8063/
 - Open the nav drawer and click "By Provider"
 - You should see a page reflecting what's shown in the designs in the attached issue above
 - Ensure that everything looks correct at both desktop and mobile resolutions
